statsmodels/stats/nodewise_regression.py

```diff
@@ -0,0 +1,124 @@
+from statsmodels.regression.linear_model import OLS
+import numpy as np
+
+
```

Comment:
On second thought, the class should be called RegularizedCovariance, and the fit method should be called "fit_nodewise"